crates/tbdex/src/http_client/exchanges.rs

```diff
@@ -151,10 +151,30 @@ pub fn get_exchange(
     Ok(exchange)
 }
 
-pub fn get_exchange_ids(pfi_did: &str, requestor_did: &BearerDid) -> Result<Vec<String>> {
+#[derive(Clone, Default, Debug, PartialEq)]
+pub struct GetExchangeIdsQueryParams {
+    pub pagination_offset: Option<i64>,
+    pub pagination_limit: Option<i64>,
+}
+
+pub fn get_exchange_ids(
+    pfi_did: &str,
+    requestor_did: &BearerDid,
+    query_params: Option<GetExchangeIdsQueryParams>,
+) -> Result<Vec<String>> {
     let service_endpoint = get_service_endpoint(pfi_did)?;
     let get_exchanges_endpoint = format!("{}/exchanges", service_endpoint);
 
+    let get_exchanges_endpoint = if let Some(params) = query_params {
+        add_pagination(
+            &get_exchanges_endpoint,
+            params.pagination_offset,
```

Comment:
Are these params specified by the client or server?

Reply:
Client. They're HTTP query params submitted by the client to query a subset of exchange IDs from the PFI.